.idea/description.html

```diff
@@ -0,0 +1 @@
+<html>Simple <b>Java</b> application that includes a class with <code>main()</code> method</html>
```

Comment:
There should be files from .idea. Add  .idea to gitignore and remove these files.